assets/package.json

```diff
@@ -76,18 +81,23 @@
     "rollup-plugin-uglify": "^6.0.4",
     "sass": "^1.26.3",
     "stylelint": "^13.2.1",
-    "stylelint-scss": "^3.15.0"
+    "stylelint-scss": "^3.15.0",
+    "uglifycss": "0.0.29"
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.12.1",
     "autotrack": "2.4.1",
     "core-js": "3.6.4",
     "d3": "5.15.0",
+    "datatables": "1.10.18",
     "date-time-format-timezone": "1.0.22",
     "element-closest": "3.0.2",
     "esri-leaflet": "2.3.3",
+    "exports-loader": "0.7.0",
     "fast-memoize": "2.5.2",
+    "jquery": "3.4.1",
```

Comment:
Why do we need to bring in jquery? I think you are bringing it in so that you can bring in datatables. Is there a reason we can't use list.js? Jquery is a very large asset and I would like to keep it out of our code.

Reply:
I deleted this from package.json.